lib/util/validator.ts

```diff
@@ -77,6 +78,8 @@ export class FieldValidator {
 
   public static readonly quoteSpeed = Joi.string().valid('fast', 'standard');
 
+  public static readonly bps = Joi.number().greater(0).max(10000); // 0 to 100%
```

Comment:
I know we set it to `10000` elsewhere but we should probably update these to the `BPS` we have in `lib/constants.ts`